ssh/tailssh/tailssh_test.go

```diff
@@ -883,7 +886,7 @@ func TestSSHAuthFlow(t *testing.T) {
 		},
 	}
 	s := &server{
-		logf: logger.Discard,
+		logf: log.Printf,
```

Comment:
Changed in order to see SSH logs during test.

Reply:
log to the `testing.T` instead then. stderr logs are super spammy and intermingled and hard to read.